controls/scripts/ControlsPublisher.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import rospy
+from geometry_msgs.msg import WrenchStamped
+from controls.msg import ThrusterForceArray
+from controls.msg import ThrusterPWMArray
+
+class ControlsPublisher:
+    """
+    Publishes controls messages describing the robot's desired motion and the required thruster commands.
+
+    The target sum of forces and torques on the robot as a whole at any given point in time are described by geometry_msgs/WrenchStampes messages published to the controls/robot/wrench topic.
+
+    The target force outputs of each thruster required to attain the target sum of forces and torques are described by controls/ThrusterForceArray messages published to the controls/thruster/forces topic.
+
+    The pwm signals required to attain each thruster's force output target are described by controls/ThrusterPWMArray messages published to the controls/thruster/pwms topic.
+
+    Attributes:
+        wrench_pub: Publisher to controls/robot/wrench. Publishes geometry_msgs/WrenchStamped messages.
+        forces_pub: Publisher to controls/thruster/forces. Publishes controls/ThrusterForceArray messages.
+        pwms_pub: Publisher to controls/thruster/pwms. Publishes controls/ThrusterPWMArray messages.
+    """
+
+    def __init__(self):
+        """Initialize ControlsPublisher instance."""
+        self.wrench_pub = rospy.Publisher('controls/robot/wrench', WrenchStamped, queue_size=10)
```

Comment:
make these snake case. @dhyang33 @krzxr  do we want to use leading underscores in python to indicate private variables? i.e. `self._wrenchPub`

Reply:
We ended up doing this.

Reply:
I think the python convention is self._wrench_pub.